straxen/scada.py

```diff
@@ -130,6 +132,11 @@ def get_scada_values(self,
                    'in utc unix time ns.')
             raise ValueError(mes)
 
+        _fill_gaps = [None, 'None', 'interpolation', 'forwardfill']
+        mes = (f'Wrong argument for "fill_gaps", must be either {_fill_gaps}.' 
+               f' You specified "{fill_gaps}"')
+        assert fill_gaps in _fill_gaps, mes
```

Comment:
perhaps a `ValueError` is more appropriate. 

I read somewhere the other day that assertion statements are usually not what you want to use because it's not compiled correctly if one ever to decides to do that. But given that I could not easily find it back, here is the python bug report:
https://bugs.python.org/issue21385